src/core/core/__init__.py

```diff
@@ -25,3 +26,4 @@ def initialize_managers(app: Flask, initial_setup: bool = False):
     api_manager.initialize(app)
     queue_manager.initialize(app, initial_setup)
     data_manager.initialize(initial_setup)
+    sentry_manager.initialize(app)
```

Comment:
should be one of the first things, to get logs if anything during initialization fails